enterprise/frontend/src/embedding-sdk/test/sdk-compatibility-warning.unit.spec.tsx

```diff
@@ -0,0 +1,116 @@
+import { render } from "@testing-library/react";
+import fetchMock from "fetch-mock";
+
+import {
+  setupCurrentUserEndpoint,
+  setupPropertiesEndpoints,
+} from "__support__/server-mocks";
+import { waitForLoaderToBeRemoved } from "__support__/ui";
+import {
+  MetabaseProvider,
+  defineEmbeddingSdkConfig,
+} from "embedding-sdk/components/public";
+import {
+  createMockSettings,
+  createMockTokenFeatures,
+  createMockUser,
+  createMockVersion,
+} from "metabase-types/api/mocks";
+
+import { getEmbeddingSdkVersion } from "../config";
+
+// TODO: extract this common setup to a shared util
```

Comment:
Now that we have more tests we're actually starting to see the patterns in these helpers so we may be ready to extract them